src/snakemake/workflow.py

```
@@ -280,6 +280,12 @@ def info_header(self):
         import hashlib
 
         conda_bin = shutil.which("conda")
+        try:
+            config_md5 = hashlib.md5(
```

Ruff (0.15.20)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)